internal/controlplane/handlers_evalstatus.go

```diff
@@ -372,6 +372,12 @@ func (s *Server) sortEntitiesEvaluationStatus(
 	profileStatuses = map[uuid.UUID]*minderv1.ProfileStatus{}
 	statusByEntity = map[string]map[uuid.UUID][]*minderv1.RuleEvaluationStatus{}
 
+	psc, err := propSvc.WithEntityCache(s.props, 1000)
```

Comment:
Do we need to allocate 1000? Why not just leave it at 0 and let the map itself do the allocation?

Reply:
it was just a number I pulled out of thin air to avoid frequent allocations. But, this handler is more DB search intensive than CPU intensive, so the allocations won't make a difference.

I dropped the prealloc in a separate patch.